ui/app/components/tool-actions-form.js

```diff
@@ -63,11 +63,15 @@ export default Component.extend(DEFAULTS, {
   dataIsEmpty: match('data', new RegExp(DEFAULTS.data)),
 
   expirationDate: computed('creation_time', 'creation_ttl', function() {
-    const { creation_time, creation_ttl } = this.getProperties('creation_time', 'creation_ttl');
+    let { creation_time, creation_ttl } = this.getProperties('creation_time', 'creation_ttl');
     if (!(creation_time && creation_ttl)) {
       return null;
     }
-    return moment(creation_time).add(moment.duration(creation_ttl, 'seconds'));
+
+    creation_time = new Date(creation_time);
+    creation_ttl = new Date(creation_ttl);
```

Comment:
I don't think we want to convert to a date here - just pass it to `addSeconds` directly.